### ISSUE TITLE
Option `addTypename` defaults to true

### ISSUE THREAD

**Issue by bearcanrun** [daugsbi/gatsby-plugin-codegen#6]
This plugin is working great autogenerating the types, but is still adding` __typename` to the declaration files. I thus get errors of "Property '__typename' is missing in type..." I can manually remove `__typename` from declarations files and all is happy again.

Apollo default for `addTypename` is true and otherwise must be unset by calling `--no-addTypename` according to cli options for [`apollo client:codegen`](https://www.npmjs.com/package/apollo#apollo-clientcodegen-output).

```
--[no-]addTypename                         [default: true] Automatically add __typename to your queries, can be unset
                                             with --no-addTypename
```

If you have a preferred method for adding this exception [here (gatsby-node.ts#L62)](https://github.com/daugsbi/gatsby-plugin-codegen/blob/0708f7b97bc8478e0dcfc1d459b85007216cbf53/src/gatsby-node.ts#L62) or elsewhere, I'd be happy to make a PR.

**Comment by bearcanrun** [comment]
I dove into your code to provide a fix and realized I was going about it wrong. Your code is correct according to Apollo docs for [creating a config](https://www.apollographql.com/docs/devtools/apollo-config/#clientaddtypename). So I'll keep looking into why `__typename` is still being output.

**Comment by daugsbi** [comment]
Hi @bearcanrun 
Thanks for bringing this up :+1:  I've checked the code and found the issue. 
The problem was, that this option was ignored in code-generation, although defined in the apollo.config.js file. I was expecting another behavior.

Now it is explicitly defined as a parameter of the cli and should work as expected. Closed by 39781ce29490e547a090df112a25c9f04330bbd3

Checkout version 1.0.14.
